operator/internal/controller/redpanda/redpanda_controller_test.go

```diff
@@ -26,7 +26,6 @@ import (
 	"github.com/go-logr/logr/testr"
 	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
 	"github.com/redpanda-data/helm-charts/pkg/kube"
-	"github.com/redpanda-data/redpanda-operator/operator/api/redpanda/v1alpha2"
```

Comment:
Do you have a linter or extension that did this cleanup automatically? 👀

Reply:
Nope, I just took a look at the warnings in VSCode and saw some lines about multiple imports of the same package with aliases, so I fixed them.

Reply:
Gotcha! I'll find us a linter to check this. I figured that goimports or staticcheck would have caught it 🤔